x-pack/plugin/src/test/resources/rest-api-spec/api/xpack.graph.explore.json

```diff
@@ -3,8 +3,8 @@
     "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/graph-explore-api.html",
     "methods": ["GET", "POST"],
     "url": {
-      "path": "/{index}/_xpack/graph/_explore",
-      "paths": ["/{index}/_xpack/graph/_explore", "/{index}/{type}/_xpack/graph/_explore"],
+      "path": "/{index}/_xpack/_graph/explore",
+      "paths": ["/{index}/_xpack/_graph/explore", "/{index}/{type}/_xpack/_graph/explore"],
       "parts" : {
```

Comment:
Remove the `_xpack`?

Reply:
@markharwood Oops, thanks! I pushed a2dc134a454d90b677580d0e185186d26eee239d.